e2e/test/scenarios/embedding/embedding-dashboard.cy.spec.js

```diff
@@ -1002,11 +1002,16 @@ H.describeEE("scenarios > embedding > dashboard appearance", () => {
       },
       {
         additionalHashOptions: {
-          locale: "de",
+          locale: "de-CH",
```

Comment:
I wonder if we have separate tests for languages codes (`de`) and language-country codes (`de-CH`) somewhere else in the codebase? Might be good to test both cases

Reply:
https://github.com/metabase/metabase/pull/51002#discussion_r1884230079
I added a lot more unit tests in that PR to test the underlying logic, this is just a check that the fallback logic is indeed used here

Reply:
Perfect, thanks!